pyrato/edc.py

```diff
@@ -67,17 +67,20 @@ def schroeder_integration(room_impulse_response, is_energy=False):
     The result is the energy decay curve for the given room impulse response
     after [#]_.
 
-    .. math:
+    .. math::
 
-        \langle e^2(t) \rangle = N\cdot \int_{t}^{\infty} h^2(\tau)
+        \langle e^2(t) \rangle = \int_{t}^{\infty} h^2(\tau)
         \mathrm{d} \tau
 
     Parameters
     ----------
-    room_impulse_response : pyfar.Signal
-        Room impulse response as array
+    room_impulse_response : pyfar.Signal, pyfar.TimeData
+        Room impulse response as pyfar audio object.
     is_energy : boolean, optional
         Whether the input represents energy data or sound pressure values.
+        By default, this is set to ``False``, in which case the upper equation
+        is applied. If set to ``True``, the function :math:`h(\tau)` is used
+        without squaring.
```

Comment:
```suggestion
        By default, this is set to ``False``. If set to ``True``, the variable :math:`h(\tau)` is
        assumed to represent energy and the power two is omitted from the given
        equation.
```
The sentence seemed a bit confusion. Maybe something along these lines?